content/docs/introduction/extra-usage.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Extra usage
+enableTableOfContents: true
+subtitle: Learn how extra usage works in Neon's pricing plans
+redirectFrom:
+  - /docs/introduction/billing-overview
+updatedOn: '2024-02-22T14:29:54.384Z'
+---
+
+Neon plans are structured around **Allowances** and **Extra usage**. Allowances are included in your plan. With Neon's paid plans, extra usage is available for a fee.
```

Comment:
"for a fee" sounds like it could be a single fee. Maybe a bit more explanation, something like:

"With Neon’s paid plans, you can purchase extra usage in set increments for when you need to go over your allowance."

Reply:
changed as suggested